answers/README.md

```diff
@@ -0,0 +1,165 @@
+# ANSWERS
+
+| Method | Route                        | Description                                                    |
+| ------ | ---------------------------- | -------------------------------------------------------------- |
+| POST   | [/answers](#post---answers)  | it will be to submit answer by the peers                       |
+| PATCH  | [/answers](#patch---answers) | for updating the answers(will primarily be using for approval) |
+| GET    | [/answers](#get---answers)   | it will be to get the answers                                  |
+
+### POST - /answers
+
+It will be used for peers to answer questions.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - **`answer=[STRING]`** required - (The answer text)
+  - **`answeredBy=[STRING]`** required (The ID of the peer who answered the question.)
+  - **`eventId=[STRING]`** required (The ID of the event where the question was asked.)
+  - **`questionId=[STRING]`** required (The ID of the question for which the answer is.)
+- **Headers**
+  - Authorization: Bearer <management_token>
```

Comment:
- what is `management_token` here?

Reply:
You can read about management token here - 
https://www.100ms.live/docs/get-started/v2/get-started/security-and-tokens#management-token-for-rest-api

Reply:
You can read about management token here - 
https://www.100ms.live/docs/get-started/v2/get-started/security-and-tokens#management-token-for-rest-api
But I think this might not be needed here